lib/data_raw.lua

```diff
@@ -1,17 +1,42 @@
 local data_raw = {}
+data_raw.removed = {}
```

Comment:
from what I gather this is both for removed and hidden things. Can this lead to problems or is the distinction unnecessary?

```suggestion
data_raw.removed_or_hidden = {}
```